src/osemgrep/cli_scan/Scan_subcommand.ml

```diff
@@ -200,6 +209,58 @@ let rules_and_counted_matches (res : Core_runner.result) : (Rule.t * int) list =
       | None -> acc)
     map []
 
+let remove_matches_in_baseline commit baseline head renamed =
```

Comment:
Please add a short comment about what this function does, and maybe add the type for the argument
with let foo (commit : ??) (baseline : ??) (head : ??) (renamed: ??) : ?? =
OCaml has type inference but it's nice for the reader to have the types in the source code.

Reply:
Done.